src/labels/CustomLabels.labels

```diff
@@ -1601,6 +1601,14 @@
         <shortDescription>Warning displayed on Customizable Rollups page when enhanced RDs are enabled</shortDescription>
         <value>&lt;b&gt;WARNING: You have Enhanced Recurring Donations enabled. If you disable Customizable Rollups, rollups to Recurring Donations will no longer function.&lt;/b&gt;</value>
     </labels>
+    <labels>
+        <fullName>RD2_ElevateClosedStatusChangeError</fullName>
```

Comment:
Let's rename the custom label to match the requirement (expected behaviour). For example: `RD2_ElevateRDClosedStatusCannotBeChanged` or `RD2_ElevateRDClosedStatusIsLocked`  or `RD2_ElevateRDStatusIsLockedWhenClosed`.